fbw-a32nx/src/wasm/fbw_a320/src/interface/SimConnectInterface.cpp

```diff
@@ -2431,6 +2431,9 @@ void SimConnectInterface::processEvent(const DWORD eventId, const DWORD data0, c
       simInputThrottles.ATHR_push = 1;
       std::cout << "WASM: event triggered: AUTO_THROTTLE_ARM" << std::endl;
       break;
+
+      // Re emitting masked event
+      execute_calculator_code("(>K:A32NX.AUTO_THROTTLE_DISCONNECT)", nullptr, nullptr, nullptr);
```

Comment:
There's the culprit for the instinctive not working with the bind. Should be on the AUTO_THROTTLE_DISCONNECT case 😄

Reply:
Ahhh damnit :D Thanks for finding that